django/db/migrations/autodetector.py

```diff
@@ -521,6 +529,9 @@ def generate_created_models(self):
                     related_fields[field.name] = field
                 if getattr(field.remote_field, "through", None) and not field.remote_field.through._meta.auto_created:
                     related_fields[field.name] = field
+            # Are there any indexes to defer?
```

Comment:
If we have indexes defined in `Meta.indexes` while creating a new model we also create `AddIndex` operations in the migration. Similar to what has been done for `AlterUniqu/IndexTogether` and `order_with_respect_to` [here](https://github.com/django/django/pull/6857/files/2496a55836b0bc941356bf04c57d31e5ecc52d6c#diff-a71b173d008afea1acd5318ef509bb7cR535).